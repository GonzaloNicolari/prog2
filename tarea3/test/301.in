# prueba de es_vacio_binario, insertar_en_binario, raiz, cantidad_binario
es_vacio_binario
cantidad_binario
insertar_en_binario (8,r)
raiz
es_vacio_binario
cantidad_binario
insertar_en_binario (6,y)
insertar_en_binario (6,t)
cantidad_binario
insertar_en_binario (4,m)
cantidad_binario
insertar_en_binario (4,g)
insertar_en_binario (4,d)
insertar_en_binario (4,h)
insertar_en_binario (4,i)
insertar_en_binario (4,j)
cantidad_binario
raiz
Fin