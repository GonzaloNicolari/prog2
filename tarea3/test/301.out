1>#  prueba de es_vacio_binario, insertar_en_binario, raiz, cantidad_binario.
2>Vacio.
3>0.
4>Insertado.
5>(8,r)
6>No vacÃ­o.
7>1.
8>Insertado.
9>Insertado.
10>3.
11>Insertado.
12>4.
13>Insertado.
14>Insertado.
15>Insertado.
16>Insertado.
17>Insertado.
18>9.
19>(8,r)
20>Fin.
